contracts/Splitter.sol

```diff
@@ -7,34 +7,29 @@ contract Splitter {
     
```

Comment:
Okay, let's start shifting our thinking about how to approach this. 

It is generally a red-flag to hard-code an address unless you are referred to something external to your own system ... e.g. UniSwap is "out there" ... otherwise, we can usually make variables of deployed addresses and the users. 

If we think of this as a general-purpose utility, then could say "Anyone acting as Alice" can split funds between two users generally referred to as "Bob and Carol" - not real people, just parties to the transaction, i.e. "roles". 

We can also make the amount flexible. The amount to split is the amount sent. 

So, the "Sender" (Alice) is `msg.sender` which is reliably whoever signed the transaction. 
The smount is `msg.value` which is reliably whatever amount of ETH was attached to the transaction. 

That's a good start. What about Bob and Carol? Who are they? They could be whoever Alice says she wants to receive the money. 

Consider:

```
function splitFunds(address recipient1, address recipient2) public payable {
  ...
}
```

Now, we have the sender, the amount and the two receivers in variables the function can play with.